cmd/new.go

```diff
@@ -57,9 +60,7 @@ func init() {
 	draftCmd.Flags().StringVar(&draftOpts.Package, "package", "", "package identifier for the release")
 	draftCmd.Flags().BoolVar(&draftOpts.NoAutoUpgrade, "no-auto-upgrade", false, "disable automatic upgrade checks [$KEYGEN_NO_AUTO_UPGRADE=1]")
 	draftCmd.Flags().StringSliceVar(&draftOpts.Entitlements, "entitlements", []string{}, "comma seperated list of entitlement constraints (e.g. --entitlements <id>,<id>,...)")
-
-	// TODO(ezekg) Prompt multi-line description input from stdin if "--"?
```

Comment:
There are 2 TODOs here. Please keep the TODO about description. Only remove the TODO regarding metadata.

Reply:
> There are 2 TODOs here. Please keep the TODO about description. Only remove the TODO regarding metadata.

Done